### PR TITLE
Parsers(ccpuk): remove oxygen_therapy duplicates

### PR DESCRIPTION
**Pull request by pipliggins**
Removed the treatment_oxygen_interface and treatment_oxygen_ mechanical_support attributes as duplicates of treatment_oxygen_therapy.

Addressing #61

### CODE REVIEW

File: isaric/schemas.py [hunk 1]
```diff
@@ -136,7 +136,6 @@ class Visit(BaseModel):
     treatment_intravenous_fluids: bool | None = None
     treatment_nsaid: bool | None = None
     treatment_neuraminidase: bool | None = None
-    treatment_oxygen_interface: bool | None = None
```
Comment (ekamau) [inline isaric/schemas.py:139]: I am of the opinion that we keep this in the schemas .. this follows my discussion with Rosario today. There might be other ISARIC datasets where there will be matching fields / columns for 'oxygen_interface' ...

File: isaric/schemas.py [hunk 2]
```diff
@@ -149,7 +148,6 @@ class Visit(BaseModel):
     treatment_monoclonal_antibody: bool | None = None
     treatment_other: list[str] = []
 
-    treatment_oxygen_mechanical_support: bool | None = None
```
Comment (ekamau) [inline isaric/schemas.py:152]: Same thing here .. There might be other ISARIC datasets where there will be matching fields / columns for 'oxygen_mechanical_support' ... 
We can remove the redundancy in the parsers, but keep the redundancy in the schemas - IMO..

**Review by ekamau — COMMENTED** [review]
changes in the CCPUK parser are ok - looks good.

For the schemas - see comments .. maybe there are other ISARIC datasets where there will be matching fields / columns for 'oxygen_interface' and 'oxygen_mechanical_support' ... just thinking ..

File: isaric/schemas.py (hunk 1 above)
Comment (pipliggins) [reply isaric/schemas.py:139]: Good point - I'll add the two oxygen treatment options back into the schema

**Pull request merged**